crates/apollo-mcp-registry/src/platform_api/operation_collections/collection_poller.rs

```diff
@@ -218,6 +275,9 @@ impl CollectionSource {
             CollectionSource::Id(id, platform_api_config) => {
                 self.collection_id_stream(id.clone(), platform_api_config.clone())
             }
+            CollectionSource::Default(graph_ref, platform_api_config) => {
+                self.default_collection_stream(graph_ref.clone(), platform_api_config.clone())
+            }
```

Comment:
If you use `ref` here and in the other match arm, you can change the signature to `into_stream(self)`:

```suggestion
            CollectionSource::Default(ref graph_ref, ref platform_api_config) => {
                self.default_collection_stream(graph_ref.clone(), platform_api_config.clone())
            }
```